modules/media/jsx/mediaIndex.js

```diff
@@ -56,6 +54,65 @@ class MediaIndex extends React.Component {
     this.refs.mediaFilter.clearFilter();
   }
 
+  /**
+   * Modify behaviour of specified column cells in the Data Table component
+   * @param {string} column - column name
+   * @param {string} cell - cell content
+   * @param {array} rowData - array of cell contents for a specific row
+   * @param {array} rowHeaders - array of table headers (column names)
+   * @return {*} a formated table cell for a given column
+   */
+  formatColumn(column, cell, rowData, rowHeaders) {
+    const hiddenHeaders = ['Cand ID', 'Session ID', 'Hide File', 'File Type'];
+
+    // If a column if set as hidden, don't display it
+    if (hiddenHeaders.indexOf(column) > -1) {
```

Comment:
@um4r12 you can do this for your hiddenheaders in https://github.com/aces/Loris/pull/3796